README.md

```diff
@@ -182,6 +182,16 @@ For example: `0001-create-users` or `0001-modify-users`
 
 In `/src/databases/seeds` there is seed or starter data. The contents can be run manually to help developers get data in their databases. The scripts are not referenced by the software and are included for convenience. The migrations must run prior to using seed scripts.
 
+We use [SQLFluff](https://docs.sqlfluff.com/en/stable/index.html)
+to lint our SQL files. See
+[Getting Started with SQLFluff](https://docs.sqlfluff.com/en/stable/gettingstarted.html)
+for instructions on setting it up.
+
+Once you have SQLFluff installed,
```

Comment:
We don't generally use this indentation style in our README -- let's keep with the current format of just having paragraphs as paragraphs in this project, if that's OK.

FWIW I do find myself reading readme files in raw form often enough that this format is notably harder for me to parse, so I also just don't really like overriding the end user's ability to just have their context determine the most appropriate wrap based on their accessibility needs (e.g. font size, screen size, etc).